### PR TITLE
Add Installations to Integration Tests

### PR DESCRIPTION
**Pull request by DellaBitta**
Add Firebase Installations to our integration test suite, and launch it as part of our Integration Test GHA Workflow.

Addtional changes to add installations to our ios_build script.

### CODE REVIEW

File: installations/integration_test/LibraryManifest.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+**
+** Unless required by applicable law or agreed to in writing, software 
+** distributed under the License is distributed on an "AS IS" BASIS, 
+** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+** See the License for the specific language governing permissions and 
+** limitations under the License.
+*/
+-->
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.google.firebase.instanceid.testapp.lib" >
```
Comment (a-maurice) [inline installations/integration_test/LibraryManifest.xml:19]: fis

File: installations/integration_test/Podfile [hunk 2]
```diff
@@ -0,0 +1,16 @@
+
+source 'https://github.com/CocoaPods/Specs.git'
+platform :ios, '8.0'
+# Firebase Instance ID test application.
```
Comment (a-maurice) [inline installations/integration_test/Podfile:4]: Installations

**Review by a-maurice — CHANGES_REQUESTED** [review]

**Review by a-maurice — APPROVED** [review]

File: installations/integration_test/Images.xcassets/AppIcon.appiconset/Contents.json [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+      "idiom" : "ios-marketing",
+      "size" : "1024x1024",
+      "scale" : "1x"
+    }
+  ],
+  "info" : {
+    "version" : 1,
+    "author" : "xcode"
+  }
+}
```
Comment (vimanyu) [inline installations/integration_test/Images.xcassets/AppIcon.appiconset/Contents.json:98]: nit: end of file newline

File: installations/integration_test/Images.xcassets/LaunchImage.launchimage/Contents.json [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+      "extent" : "full-screen",
+      "minimum-system-version" : "7.0",
+      "scale" : "2x"
+    }
+  ],
+  "info" : {
+    "version" : 1,
+    "author" : "xcode"
+  }
+}
```
Comment (vimanyu) [inline installations/integration_test/Images.xcassets/LaunchImage.launchimage/Contents.json:51]: nit: end of file newline

File: installations/integration_test/build.gradle [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+  // If this is running form inside the SDK directory, run the setup script.
+  if (project.file('../../setup_integration_tests.py').exists()) {
+    commandLine 'python', '../../setup_integration_tests.py', project.projectDir.toString()
+  }
+  else {
+    commandLine 'echo', ''
+  }
+}
+
+build.dependsOn(copyIntegrationTestFiles)
```
Comment (vimanyu) [inline installations/integration_test/build.gradle:90]: nit: end of file newline

File: installations/integration_test/settings.gradle [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+    }
+  }
+}
+if (!(new File(firebase_cpp_sdk_dir)).exists()) {
+  throw new StopActionException(
+    sprintf('Firebase C++ SDK directory %s does not exist',
+            firebase_cpp_sdk_dir))
+}
+gradle.ext.firebase_cpp_sdk_dir = "$firebase_cpp_sdk_dir"
+includeBuild "$firebase_cpp_sdk_dir"
```
Comment (vimanyu) [inline installations/integration_test/settings.gradle:39]: nit: end of file newline

File: installations/integration_test/src/integration_test.cc [hunk 7]
```diff
@@ -0,0 +1,232 @@
...
+  firebase::Future<std::string> token = installations_->GetToken(false);
+  WaitForCompletion(token, "GetToken");
+  EXPECT_NE(*token.result(), "");
+  std::string first_token = *token.result();
+  token = installations_->GetToken(true);
+  WaitForCompletion(token, "GetToken 2");
+  EXPECT_NE(*token.result(), first_token);
+}
+
+}  // namespace firebase_testapp_automated
```
Comment (vimanyu) [inline installations/integration_test/src/integration_test.cc:232]: nit: end of file newline

**Review by vimanyu — COMMENTED** [review]
nit: missing end of file newlines.
Other than that, LGTM!

**Review by a-maurice — APPROVED** [review]

**Review by anonymous-akorn — APPROVED** [review]
Integration into the test infrastructure looks good AFAICT. Most recent run on GHA shows it passing.

**Pull request merged**